typescript/scheduledTasks.ts

```diff
@@ -9,16 +8,9 @@ export type TaskAction = BaseTaskAction &
         type: 'FORM'
         formId: number
       }
-    | {
-        type: 'HREF'
-        link: {
-          text: string
-          url: string
-        }
-      }
     | {
         type: 'CHANGE_STATUS'
-        status: TaskActionStatus
+        status: 'COMPLETE'
```

Comment:
Can this `type` just be `COMPLETE`? What are we trying to cater for in the future?

Reply:
If this is how we are going to consider there to be an action to complete a task, we prob need to make actions required and validate that at least one action is set, otherwise we will not know how to complete a task.

Reply:
I'm trying to cater for a best guess of what we might see in the future. If we set it the type to `COMPLETE` we don't really gain anything, but introduce a stronger possibility of needing to change the data structure

Reply:
IMO, we just only allow tasks with actions to be assigned to apps, and we set both tasks in the console when the user selects a form for their task

Reply:
I think I've changed my mind on this, and I want to make `actions` required.